### PR TITLE
Add map and foursquare

### PR DESCRIPTION
**Pull request by a-ashbeck**

### CODE REVIEW

File: foursquareApiKey.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (cbrenner04) [inline foursquareApiKey.js]: you may want to remove this

Comment (a-ashbeck) [reply foursquareApiKey.js]: is aware. is won't leave. is gave up.

File: public/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="/css/bootstrap.min.css">
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
     <link rel="stylesheet" type="text/css" href="./css/application.css">
+    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?libraries=visualization&key=__API_KEY_HERE__"></script>
```
Comment (cbrenner04) [inline public/index.html]: how does this work? what is  `___API_KEY_HERE___` ?

Comment (a-ashbeck) [reply public/index.html]: no magic, you just have to copy and paste the key in there

File: app/components/Places.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      return (
+        <li key={i}>{venue.name}</li>
+      )
+    })
+
+    return (
+      <div>
+        <h1>Venues</h1>
+        <ol>
+          {list}
```
Comment (cbrenner04) [inline app/components/Places.js]: where does `{list}` come from?

File: public/index.html (hunk 2 above)
Comment (cbrenner04) [reply public/index.html]: i don't understand.. what do we do when we deploy it?

File: foursquareApiKey.js (hunk 1 above)
Comment (cbrenner04) [reply foursquareApiKey.js]: ?

File: public/index.html (hunk 2 above)
Comment (a-ashbeck) [reply public/index.html]: we'll leave it in. client side google api keys can't be hidden, but that's ok. we just need to generate a new one later that specifies which hosts can make requests with it.

Comment (cbrenner04) [reply public/index.html]: cool

**Review by cbrenner04 — APPROVED** [review]

**Pull request merged**